test/Project.toml

```diff
@@ -2,5 +2,16 @@
 CUDA = "052768ef-5323-5732-b1bb-66c8b64840ba"
 Flux = "587475ba-b771-5e3f-ad9e-33799f191a9c"
 Images = "916415d5-f1e6-5110-898d-aaa5f9f070e0"
-ReTest = "e0db7c4e-2690-44b9-bad6-7687da720f89"
+ReTestItems = "817f1d60-ba6b-4fd5-9520-3cf149f6a823"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+TestItems = "1c621080-faea-4a02-84b6-bbd5e436b8fe"
```

Comment:
My impression is that we don't need TestItems if we have ReTestItems?

Reply:
I figured that out after CI started complaining :)

The big problem with ReTestItems right now is that it doesn't support 1.6. My thought is to switch to TestItemRunner.jl, but that would mean we miss out on parallel execution (most importantly parallel _compilation_) for Buildkite. If https://github.com/JuliaTesting/ReTestItems.jl/issues/23 is addressed, perhaps we can revisit.

Reply:
We could use TestItemRunner for 1.6 and ReTestItems for the rest?